holidays/financial/ny_stock_exchange.py

```diff
@@ -245,4 +245,5 @@ class NewYorkStockExchangeStaticHolidays:
             (OCT, 30, "Hurricane Sandy"),
         ),
         2018: (DEC, 5, "Day of Mourning for President George H.W. Bush"),
+        2025: (JAN, 9, "Day of Mourning for President Jimmy Carter"),
```

Comment:
I'm not very familiar with the American tradition, but maybe "Day of Mourning for President James E. Carter" (as for other presidents)?

Reply:
His [Presidential Library website](https://www.jimmycarterlibrary.gov/) didn't mention his middle name either so the current one looks fine to me IMO, neither does [Ronald Reagan](https://www.reaganlibrary.gov/) nor [Barrack Obama](https://obamalibrary.archives.gov/#event-/timeline/item/presidential-inauguration).

Reply:
Yes, there is a need for some standardization here. It seems the official name should include at least "National Day of Mourning" with additional details with no clear standardization way.

Let's use it as is for now and create an issue to address this later.

Reply:
Please use https://github.com/vacanza/holidays/issues/2203 for discussion.